packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -17,6 +17,7 @@ public struct MediaInfo: Encodable {
 /// Definition of capabilities to enable in the Block Editor
 public enum Capabilities: String {
     case contactInfoBlock
+    case layoutGridBlock
     case mediaFilesCollectionBlock
```

Comment:
@mchowning @hypest I'd be curious to get your thoughts on these three capabilities (`contactInfoBlock`, `mediaFilesCollectionBlock`, and `layoutGridBlock`). This becomes a not so secretive passthrough for  `jetpack` content. I wonder if we should make this more universal by changing this capability to something like `additionalSupportedBlocks` that contains an array of the block names the client wants to support. 

`audioBlock` might be a good candidate to include in this as well. WDYT? 

--- 

To be clear @enejb is following the established pattern here and incorporating my change suggestion would increase the scope of this PR so I'm happy to take that on under a different PR. I wanted to see if there was any existing context or if it's just the evolution of the architecture.

Reply:
> I wonder if we should make this more universal by changing this capability to something like additionalSupportedBlocks that contains an array of the block names the client wants to support.

I agree. I think it's fine for this PR, but this is something we should fix.

Reply:
Agreed as well. I think it's easier to see this when you have enough occurrences that it starts to become a pattern. That's the right time to make an abstraction with the characteristics that are common to all 👍

Reply:
+1 to what the fine folks above be already said, for generalizing/abstracting the "block offerability" flags.